plugins/README.md

```diff
@@ -0,0 +1,246 @@
+	},
+	"scripts": {
+		"build": "wp-scripts build",
+		"plugin-zip": "wp-scripts plugin-zip",
+		"start": "wp-scripts start"
+	}
+}
+```
+
+## Block Plugins
+
+Block plugins should follow the [block specifec plugin guidelines](https://developer.wordpress.org/plugins/wordpress-org/block-specific-plugin-guidelines/#block-plugins-and-the-block-directory) and contain only one top-level block per plugin.
+
+The block name should use the prefix `gutenberg` (e.g. `gutenberg/block-name` ) to differentiate it from core blocks.
+
+### block.json
+
+Block plugins should define a `block.json` file. Currently the `block.json` file should be located in `plugins/PLUGIN-SLUG/block.json`, rather than `plugins/PLUGIN-SLUG/src/block.json`, so it will be available in the Github PHPUnit test environment for block registration. Plugins should use the most recent `block.json` version supported by the current WordPress release.
+
+`block.json` files for plugins have some differeneces from those for core blocks (from `@wordpress/block-library`)
+
+- The `__experimental:` property should not be used, as it has no effect for block plugins.
+- Properties that specify file paths are required, unlike with core blocks: `"editorScript"`, `"script"`, `"viewScript"`, `"viewScriptModule"`, `"editorStyle"`, `"style"`, and `"viewStyle"` properties should be defined explictly, when needed, as they are not infered automatically.
+
+## Development
+
+### Scripts
+
+Top level commands like `npm install`, `npm run build`, `npm run lint` will automatically run the appropriate scripts for all plugins.
+
+`plugins/*` is defined as an [npm workspace](https://docs.npmjs.com/cli/v10/using-npm/workspaces), to support running commands for individual plugins.
+
+For example, `npm run build -w plugins/PLUGIN-SLUG` will build only the specified plugin. Check the plugin's package.json file to see which commands are supported.
+
+### wp-env
+
+New plugins should be added to `.wp-env.json` so that they are available in the wp-env docker environments for development and end-to-end testing.
+
+```json
+{
+	...
+	"plugins": [
+		".",
+		"./plugins/PLUGIN-SLUG"
+	],
+	...
+}
+```
+
+### Linting
+
+Plugin linting and formatting for PHP, JavaScript, CSS, etc is integrated into the main repository scripts, e.g. `npm run lint`.
+
+#### PHPCS
+
+Plugins need their own PHPCS configuration file that inherits shared rules and overrides settings for the text domain:
+
+`plugins/PLUGIN-SLUG/phpcs.xml.dist`
+
+```xml
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for the PLUGIN_NAME plugin">
+	<description>Includes shared sniffs for Gutenberg, with modifications.</description>
+
+	<!-- Import shared rules for monorepo -->
+	<rule ref="../../phpcs.xml.shared"/>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<property name="text_domain" type="array">
+				<element value="PLUGIN-SLUG"/>
+			</property>
+		</properties>
+	</rule>
+
+	<file>./</file>
+</ruleset>
+```
+
+#### ESLint
```

Comment:
Can this override be done in an `.eslintrc.js` within the plugin folder?

Reply:
I can be, and that's what I had at first, though I changed it on based on @gziolo 's feedback (https://github.com/WordPress/gutenberg/pull/61504#discussion_r1596342389)

Reply:
Personally I feel collocating is better as most people will just copy/paste a plugin folder and adapt it. Simpler than having to change files elsewhere. 

That said, it's not a big deal either if it stays as is.